gae/article.tpl

```diff
@@ -5,9 +5,10 @@
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,400italic,500,500italic,700,700italic|Roboto+Mono:400,500,700|Material+Icons">
     <link rel="stylesheet" href="/wf-local/styles/devsite-google-blue.css">
     <meta name="robots" content="noindex">
+    <link rel="chrome-webstore-item" href="https://chrome.google.com/webstore/detail/blipmdconlkpinefehnmjammfjpmpbjk">
```

Comment:
This file is only used for our staging environment - do you really want to add this here as well? Just confirming.